tap_mavenlink/streams/custom_field_sets.py

```diff
@@ -0,0 +1,19 @@
+from tap_mavenlink.streams.base import BaseStream
+import singer
+
+LOGGER = singer.get_logger()  # noqa
```

Comment:
What does `# noqa` mean on these?

Reply:
`# noqa` usually means that linters should ignore that line, although we're not running linters as part of any checks right now :shrug: In this case it's a bit of a copy-paste survivor from other files / lines.

Reply:
Nice... Learn something new :D

Reply:
+1